db/tests/types/fixtures.py

```diff
@@ -0,0 +1,31 @@
+import pytest
```

Comment:
Please add comments in the file indicating that they should not be used for testing `base` or `install`.

Reply:
Sure.